docs/Configuration/Core-Configuration/Alternate-Channel-Storage-Backends.md

```diff
@@ -32,6 +39,33 @@ channel_storage_backend = ao2_legacy ; Select the channel storage backend
 Before using the alternate backend in production, you should thoroughly test using your typical configuration and workload.  If you deploy to multiple production Asterisk instances, we'd also suggest slow rolling the use of the alternate backend.
 ///
 
+### Important
+
+> **IMPORTANT**
+> This configuration **must** be set before Asterisk starts, as the backend is selected during core initialization.
+> A **full restart** of Asterisk is required after making changes to `channel_storage_backend`.
+> Reloading the configuration is **not** sufficient.
+
```

Comment:
I'd move this up right after the asterisk.conf snippet.

Remove the two "Important" lines and the `>`s then surround the paragraph with a warning admonition...

```
/// warning
This configuration **must** be set before Asterisk starts, as the backend is
selected during core initialization.  A **full restart** of Asterisk is required
after making changes to `channel_storage_backend`. Reloading the configuration is
**not** sufficient.
///
```